### DISCUSSION TITLE
Debian 11 Image

### DISCUSSION THREAD

**Discussion by chrisgo** [utmapp/UTM#3550] [UTM Help (macOS)]
Is this the place to try to request a Debian 11 (minimal) image?  If not, how would you go about making this?

Trying to run Vagrant + Debian 11 on Mac M1 chip

Maybe this is the only way to do it for now?
https://www.coredump.id.au/debian-arm64-and-utm-on-apple-m1-macs/

**Comment by osy** [comment]
Do they have an ISO installer image? If so I imagine the steps will be similar to Ubuntu.

**Comment by aleksandrgilfanov** [comment]
Original iso https://cdimage.debian.org/debian-cd/current/arm64/iso-cd/ works fine in UTM.
